test/forwarddiff.jl

```diff
@@ -216,7 +258,7 @@ end
 
     function compute_nth_derivative(n, f, x)
         (n == 0) && return f(x)
-        ForwardDiff.derivative(x -> compute_nth_derivative(n - 1, f, x), x)
+        tagged_derivative(x -> compute_nth_derivative(n - 1, f, x), x)
```

Comment:
In this test we have recursion for n-th order derivatives with ForwardDiff, probably with fixed tag this has perturbation confusion?